src/ZiraLink.Api.Application/Services/CustomerService.cs

```diff
@@ -91,16 +87,20 @@ public async Task<Guid> CreateAsync(string username, string password, string ema
                 Name = name,
                 Family = family
             };
-            var content = new StringContent(JsonSerializer.Serialize(jsonObject), Encoding.UTF8, "application/json");
-            var baseUri = _idsUri;
-            var uri = new Uri(baseUri, "User");
-            var response = await client.PostAsync(uri.ToString(), content);
 
-            var responseString = await response.Content.ReadAsStringAsync(cancellationToken);
-            var userCreationResult = JsonSerializer.Deserialize<ApiResponse<string>>(responseString, new JsonSerializerOptions { PropertyNamingPolicy = JsonNamingPolicy.CamelCase });
+            // var content = new StringContent(JsonSerializer.Serialize(jsonObject), Encoding.UTF8, "application/json");
+            // var baseUri = _idsUri;
+            // var uri = new Uri(baseUri, "User");
+            // var response = await client.PostAsync(uri.ToString(), content);
+
+            // var responseString = await response.Content.ReadAsStringAsync(cancellationToken);
+            // var userCreationResult = JsonSerializer.Deserialize<ApiResponse<string>>(responseString, new JsonSerializerOptions { PropertyNamingPolicy = JsonNamingPolicy.CamelCase });
+
+            var userCreationResult = await _httpTools.CallIDSApis<ApiResponse<string>>(httpClient, "User", jsonObject, HttpMethod.Post, cancellationToken);
```

Comment:
I would create an HTTP client for IDS with all the required methods like CreateUser, ChangePassword, and ... This is hard to understand. I recommend creating the required methods and passing the parameters instead of a single object.

Reply:
Done.